src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -938,10 +1034,35 @@ internal void WriteLine(string format, object arg1, object arg2)
                     PSTraceSourceOptions.WriteLine,
                     writeLineFormatter,
                     format,
-                    new object[] { arg1, arg2 });
+                    arg1, arg2);
             }
         }
 
+        internal void WriteLine(string format, string arg1, string arg2)
+        {
+            WriteLine(format, (object)arg1, (object)arg2);
+        }
```

Comment:
why do we need this overload? won't `WriteLine(string format, object arg1, object arg2)` be sufficient for this case?

Reply:
It is more easily to understand a developer intention.
And it is one step to removing `params object[] args` oveloads although it is unclear how to do it because String.Format() (StringBuilder.AddperdFormar()) based on `params object[] args`.